packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -27,22 +27,9 @@ const WrappedNavigationMenuSelector = ( {
 	createNavigationMenuIsError,
 } ) => (
 	<NavigationMenuSelector
+		convertClassicMenu={ convertClassicMenu }
```

Comment:
Why remove the `on` prefix? It's an event for which we offer a handle.

Reply:
I think because its meaning has changed. `convertClassicMenu` is literally the function which does the conversion rather than an event handler.